### PR TITLE
根据设计稿，修复ui细节

### PR DESCRIPTION
**Pull request by yanzhangpeng**
# Description

_Please explain the changes you've made_
Modify the details of the Overview function ui

## Issue reference

We strive to have all PR being opened based on an issue, where the problem or feature have been discussed prior to implementation.

Please reference the issue this PR will close: #_[issue number]_

## Checklist

Please make sure you've completed the relevant tasks for this PR, out of the following list:

* [ ] Code compiles correctly
* [ ] Created/updated tests
* [ ] Extended the documentation

### CODE REVIEW

**Comment by doddgu** [comment]
@codding-y PTAL

File: src/Web/Masa.Dcc.Web.Admin/Masa.Dcc.Web.Admin.Rcl/Pages/App.razor [hunk 1]
```diff
@@ -1,126 +1,129 @@
...
-                <MRow NoGutters Class="body2">
-                    @_projectDetail.Identity
-                </MRow>
-                <MRow NoGutters Class="mt-2 body2">
-                    @T("AppCount")：<span class="subtitle2">@_apps.Count</span>
-                </MRow>
-                <div style="height: calc(100vh - 320px); overflow-y: auto; overflow-x: hidden;">
-                    <MRow NoGutters Class="mt-4 body2" Style="word-break: break-all;">
-                        @_projectDetail.Description
+<MRow style="">
```
Comment (MayueCif) [inline src/Web/Masa.Dcc.Web.Admin/Masa.Dcc.Web.Admin.Rcl/Pages/App.razor:4]: style=""

File: src/Web/Masa.Dcc.Web.Admin/Masa.Dcc.Web.Admin.Rcl/Pages/App.razor [hunk 2]
```diff
@@ -1,126 +1,129 @@
...
-                    <div class="emphasis2--text h7" @onclick:stopPropagation>
-                        @if (!_isEditBiz)
-                        {
-                            @_bizDetail.Name
-                        }
-                        else
+        </MNavigationDrawer>
+    </MCol>
+    <MCol Cols="10">
+        <div>
```
Comment (MayueCif) [inline src/Web/Masa.Dcc.Web.Admin/Masa.Dcc.Web.Admin.Rcl/Pages/App.razor:45]: delete this div

File: src/Web/Masa.Dcc.Web.Admin/Masa.Dcc.Web.Admin.Rcl/Pages/Overview.razor [hunk 3]
```diff
@@ -29,42 +29,40 @@
...
-                                        </MTooltip>
-                                }
-                            </MListItemGroup>
-                        </MList>
-                    </div>
-                </MNavigationDrawer>
-
-                <div class="pl-6" style="width: calc(100% - 300px); display:block;">
-                    <div style="overflow: hidden;">
+        <MTabItem Class="dd">
```
Comment (MayueCif) [inline src/Web/Masa.Dcc.Web.Admin/Masa.Dcc.Web.Admin.Rcl/Pages/Overview.razor:32]: delete Class="dd"

File: src/Web/Masa.Dcc.Web.Admin/Masa.Dcc.Web.Admin.Rcl/Pages/Overview.razor [hunk 4]
```diff
@@ -29,42 +29,40 @@
...
+            <MRow Style="">
```
Comment (MayueCif) [inline src/Web/Masa.Dcc.Web.Admin/Masa.Dcc.Web.Admin.Rcl/Pages/Overview.razor:33]: Style=""

**Review by MayueCif — APPROVED** [review]

**Pull request merged**